### PR TITLE
i18n: Update zh-CN [ Chinese (China) ] translations.

### PR DESCRIPTION
**Pull request by KenOokamiHoro**

### CODE REVIEW

**Review by PeterCxy — APPROVED** [review]
Translations are generally acceptable.

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -207,6 +225,18 @@ zh-CN:
...
     mutes: 你所静音的用户
     storage: 媒体容量大小
+  followers:
+    domain: 域名
+    explanation_html: 想要保护你的嘟文的话，请慎重考虑关注你的人。<strong>你的受保护的嘟文会发送到有你的关注者的所有实例上</strong>。你也许想要复查一下关注者列表来移除那些你无法信任的关注者。
+    followers_count: 关注者数量
+    lock_link: 保护你的账户
+    purge: 从关注者中移除
+    success:
+      one: In the process of soft-blocking followers from one domain...
```
Comment (Artoria2e5) [inline config/locales/zh-CN.yml]: Remove the `one` line as Chinese won't use it anyway.

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ zh-CN:
     about_this: 关于本实例
     apps: 应用程序
     business_email: 商业电邮︰
+    closed_registrations: 这个实例暂时（或者永久）不开放注册 _(:3」∠)_
```
Comment (Artoria2e5) [inline config/locales/zh-CN.yml]: I do not like the idea of "temporarily (or permanently)". How about "目前" (currently)? Let's not try to probe into the future...

File: app/views/user_mailer/reset_password_instructions.zh-cn.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<p><%= @resource.email %> ，嗨呀!!</p>
+
+<p>有人（可能是你）请求更改你 Mastodon 账户的密码。如果是你的话，请点击下面的链接：</p>
+
+<p><%= link_to 'Change my password', edit_password_url(@resource, reset_password_token: @token) %></p>
```
Comment (Artoria2e5) [inline app/views/user_mailer/reset_password_instructions.zh-cn.html.erb]: Change my password -> 更改密码

File: app/views/user_mailer/confirmation_instructions.zh-cn.text.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<%= @resource.email %> ，嗨呀!
+
+你刚刚在 <%= @instance %> 创建了帐号。
+
+点击下面的链接来完成注册啦 : <br>
+<%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %>
```
Comment (Artoria2e5) [inline app/views/user_mailer/confirmation_instructions.zh-cn.text.erb]: Confirm my account -> 确认帐号

File: app/views/user_mailer/confirmation_instructions.zh-cn.text.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+别忘了看看 <%= link_to 'terms and conditions', terms_url %>。
```
Comment (Artoria2e5) [inline app/views/user_mailer/confirmation_instructions.zh-cn.text.erb]: missing translation, 使用条款

**Review by Artoria2e5 — CHANGES_REQUESTED** [review]
Mostly good. Awoo.

File: config/locales/doorkeeper.zh-CN.yml [hunk 6]
```diff
@@ -15,7 +15,6 @@ zh-CN:
               invalid_uri: '必须是有效的 URL 格式'
               relative_uri: '必须是绝对的 URL 地址'
               secured_uri: '必须是 HTTPS/SSL 的 URL 地址'
-
```
Comment (Artoria2e5) [inline config/locales/doorkeeper.zh-CN.yml]: Ahem..

**Comment by KenOokamiHoro** [comment]
😌…… & @Artoria2e5 😅

File: app/views/user_mailer/reset_password_instructions.zh-cn.html.erb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<p><%= @resource.email %> ，嗨呀!!</p>
+
+<p>有人（可能是你）请求更改你 Mastodon 账户的密码。如果是你的话，请点击下面的链接：</p>
```
Comment (Artoria2e5) [inline app/views/user_mailer/reset_password_instructions.zh-cn.html.erb]: What about "hopefully you"（但愿是你）?

**Review by KenOokamiHoro — COMMENTED** [review]
awoo~

File: app/javascript/mastodon/locales/zh-CN.json [hunk 8]
```diff
@@ -77,9 +76,11 @@
   "navigation_bar.edit_profile": "修改个人资料",
   "navigation_bar.favourites": "赞的内容",
   "navigation_bar.follow_requests": "关注请求",
-  "navigation_bar.info": "关于本服务站",
+  "navigation_bar.info": "追加信息",
   "navigation_bar.logout": "注销",
-  "navigation_bar.mutes": "Muted users",
+  "navigation_bar.mutes": "被静音的用户",
+  // intentional departure from https://github.com/tootsuite/mastodon/blob/f864fee1/config/locales/zh-CN.yml#L126:
```
Comment (Artoria2e5) [inline app/javascript/mastodon/locales/zh-CN.json]: Why the change to json? The zh-CN file used to have some react magic inherited from zh-HK that has to be run as actual executable code. And json does not play well with comments.

File: app/views/user_mailer/reset_password_instructions.zh-cn.html.erb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<p><%= @resource.email %> ，嗨呀!!</p>
+
+<p>有人（但愿是你 ~ ）请求更改你 Mastodon 账户的密码。如果是你的话，请点击下面的链接：</p>
```
Comment (Artoria2e5) [inline app/views/user_mailer/reset_password_instructions.zh-cn.html.erb]: Please, no tildes, no ASCII tildes, and no spaces around tildes or before a closing parenthesis.

File: app/javascript/mastodon/locales/zh-CN.json [hunk 10]
```diff
@@ -159,5 +158,7 @@
   "video_player.expand": "展开影片",
   "video_player.toggle_sound": "开关音效",
   "video_player.toggle_visible": "打开或关上",
-  "video_player.video_error": "Video could not be played"
-}
\ No newline at end of file
+  "video_player.video_error": "视频无法播放啦……",
```
Comment (Gargron) [inline app/javascript/mastodon/locales/zh-CN.json]: This is a JSON file now, so no trailing comma, and nothing after the closing }

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/doorkeeper.zh-CN.yml [hunk 11]
```diff
@@ -124,4 +123,9 @@ zh-CN:
           home: '首页'
       application:
         title: 'OAuth 认证'
+     scopes:
```
Comment (Gargron) [inline config/locales/doorkeeper.zh-CN.yml]: There seems to be some sort of YML syntax error in this file, but I am not sure which exactly.

`I18n::InvalidLocaleData: can not load translations from /home/travis/build/tootsuite/mastodon/config/locales/doorkeeper.zh-CN.yml: #<Psych::SyntaxError: (/home/travis/build/tootsuite/mastodon/config/locales/doorkeeper.zh-CN.yml): did not find expected key while parsing a block mapping at line 19 column 5>`

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/doorkeeper.zh-CN.yml (hunk 11 above)
Comment (KenOokamiHoro) [inline config/locales/doorkeeper.zh-CN.yml]: unexpected indent ? 😂

**Comment by Gargron** [comment]
Looks like the last failing part is these unused keys:

```
Unused keys (3) | i18n-tasks v0.9.13

+--------+------------------------------+--------------------------------+

| Locale | Key                          | Value                          |

+--------+------------------------------+--------------------------------+

| zh-CN  | media_attachments            |                                |

| zh-CN  | validations.images_and_video | 已经有照片的嘟文里没法加入视频 |

| zh-CN  | validations.too_many         | 文件太多（>4）                 |

+--------+------------------------------+--------------------------------+
```

**Review by Gargron — APPROVED** [review]

**Comment by Artoria2e5** [comment]
Hmm, I will probably change all the 账 to 帐 -- Mastodon is not a banking app after all.

**Pull request merged**